File: resolucoes/python/lista01/ex015.py
# Crie um programa que leia o número de dias trabalhados em um mês e mostre o salário de um funcionário, sabendo que ele trabalha 8 horas por dia e ganha R$25 por hora trabalhada.

dias_trabalhados = int(input('Digite a quantidade de dias trabalhados: '))

salario = dias_trabalhados * (25 * 8)

print(f'O salário desse funcionário é de: R${salario}')

File: resolucoes/TypeScript/lista01/ex015.ts
import prompt from "prompt-sync"

export function main() {
    const p = prompt()

    const dias = Number(p("Quantos dias trabalhados?"))

    const horas = 8
    const ganha = 25

    const total = (horas * ganha) * dias
    console.log(`Seu salario é ${total}`)

}